kafka-ui-react-app/src/lib/hooks/__tests__/dateTimeHelpers.spec.ts

```diff
@@ -12,12 +12,20 @@ describe('dateTimeHelpers', () => {
 
     it('should output the correct date', () => {
       const date = new Date();
-      expect(formatTimestamp(date)).toBe(
-        date.toLocaleString([], { hour12: false })
-      );
-      expect(formatTimestamp(date.getTime())).toBe(
-        date.toLocaleString([], { hour12: false })
-      );
+      const formattedDate = formatTimestamp(date);
+      const formattedTimestamp = formatTimestamp(date.getTime());
+
+      const expected = `${
```

Comment:
since the default parameter of formatTimestamp is `{ hourCycle: 'h23' }` , what if instead of constructing it , we can just `date.toLocaleString([], { hourCycle: 'h23' })` , did you try this way ?

Reply:
@MikeStrike101 can you take a look?

Reply:
@Haarolean soory I was on vacation these days. Will take a look now.